playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java

```diff
@@ -396,10 +399,10 @@ private MediaSource buildInternalExoMediaSource(PKMediaSourceConfig sourceConfig
             // mp4 and mp3 both use ExtractorMediaSource
             case mp4:
             case mp3:
+            case udp:
                 mediaSource = new ProgressiveMediaSource.Factory(dataSourceFactory)
```

Comment:
Seems we need to use ExtractorMediaSource
https://medium.com/@jatinmishra27/playing-udp-stream-in-exoplayer2-55ea3ea6e1db